projects/fsastorefrontstyles/scss/components/checkout/_mini-cart.scss

```diff
@@ -31,3 +31,37 @@
     }
   }
 }
+
+.action-wrapper {
```

Comment:
We should have a context class name here, because as far as I can see this cannot be reused anywhere else, because it's pretty specific. Maybe `.coupon-sync-pilot` 🙂 ?